jupyterhub/apihandlers/users.py

```diff
@@ -237,6 +237,12 @@ async def delete(self, name):
                 )
 
         await maybe_future(self.authenticator.delete_user(user))
+
+        # Delete the user's storage, if it exists.
+        # In kubespawner spawner.py, we have delete_forever
+        # method to delete the user's PVC.
+        user.spawner.delete_forever()
```

Comment:
This should match the `await maybe_future()` above to allow async operations. No need for kubespawner discussion in the comments. If there's a comment, use "allow the spawner to cleanup any persistent resources associated with the user.

Probably even want try/except here because I suspect we don't want a spawner operation error to prevent user deletion.

Reply:
added 'await' and try/except, also changed comment